lib/chef-dk/skeletons/code_generator/files/default/delivery-project.toml

```diff
@@ -0,0 +1,21 @@
+# Delivery Prototype for Local Phases Execution
+#
+# The purpose of this file is to prototype a new way to execute
+# phases locally on your workstation. The delivery-cli will read
+# this file and execute the command(s) that are configured for
+# each phase. You can customize them by just modifying the phase
+# key on this file.
+#
+# By default these phases are configured for Cookbook Workflow only
+#
+# As this is still a prototype we are not modifying the current
+# config.json file and it will continue working as usual. 
+
+[local_phases]
+unit = "rspec spec/"
+lint = "cookstyle --with-flags"
```

Comment:
😂 I don't think that flag actually exist.. I will remove it but, what flags do we use? I looked at delivery-cli and it just have `cookstyle` without args.. Comments?